### ISSUE TITLE
Broken thumbnails appear as text

### ISSUE THREAD

**Issue by TravisSpomer** [microsoft/ficus#9]
There's no fallback for Figma file thumbnails, so they just appear as alt text, which looks quite bad. Missing and broken thumbnails should use either a placeholder image or just be removed entirely.
